src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -166,17 +164,15 @@ internal static class CodeActionHelpers
         {
             var actionSets = await GetActionSetsAsync(
                 document, codeFixService, codeRefactoringService, selection, cancellationToken).ConfigureAwait(false);
-            if (!actionSets.HasValue)
+            if (actionSets.IsDefaultOrEmpty)
             {
                 actionSets = await GetActionSetsAsync(
                     document, codeFixService, codeRefactoringService, selection, cancellationToken).ConfigureAwait(false);
 
-                if (!actionSets.HasValue)
-                {
+                if (actionSets.IsDefaultOrEmpty)
                     return ImmutableArray<CodeAction>.Empty;
-                }
```

Comment:
note: thsi code really needs a comment.  it's completely unclear why we're computing the same value twice.  that shoulnd't make any difference.  i'm leaving this as is, but i need info here @allisonchou about what's going on.

Reply:
Whoops! I have no idea how that made it in. 😬 I can submit a follow up PR to fix this once this PR is merged, thanks for the catch.